src/amo/components/UserReview/index.js

```diff
@@ -76,8 +79,7 @@ export const UserReviewBase = (props: InternalProps) => {
     showRating = false,
   } = props;
 
-  const showMoreCardId =
-    review && review.id ? String(review.id) : 'loading-text';
+  const showMoreCardId = review && review.id ? String(review.id) : loadingId;
```

Comment:
Interesting that you have to cast `review.id` to a `string`. I guess Flow doesn't like it but `contentId` is defined to accept both, so it should be possible without a cast. This is why I think it's better to restrict the `contentId` type: there is no choice to make, it's `'some-string'` or `String(id)`, and nothing else.

Reply:
Except for when it's null, which I think happens with `Addon`. Also, the reason it is being cast to a string there isn't because of `childId`, but because of the `ShowMoreCard`'s `id`.

My issue with changing `childId` to require a string is that then we'd have to cast in `Addon` and in `PermissionsCard`, and probably in anything in the future that uses it, and there's really no reason to require a string inside `ShowMoreCard`.

Reply:
To be clear, my comment does not prevent you from landing this patch.